python/src/aiconfig/editor/client/src/LocalEditor.tsx

```diff
@@ -29,6 +26,31 @@ export default function Editor() {
     loadConfig();
   }, [loadConfig]);
 
+  const enableTelemetryIfAllowed = useCallback(async () => {
+    const res = await ufetch.post(ROUTE_TABLE.LOAD, {}); // change this to the endpoint used to get the config
+
+    const enableTelemetry = res.allow_usage_data_sharing;
+
+    if (enableTelemetry) {
+      datadogLogs.init({
+        clientToken: "pub356987caf022337989e492681d1944a8",
+        env: process.env.NODE_ENV ?? "development",
+        service: "aiconfig-editor",
+        site: "us5.datadoghq.com",
+        forwardErrorsToLogs: true,
+        sessionSampleRate: 100,
```

Comment:
In diff summary, can you link to website for API docs so we can read more on what these mean?

Reply:
whoops forgot to link docs. Generally, datadog docs are easily googleable and easily navigate able. Can just google "datadog <search term>" 

"datadog browser logging" ->  https://docs.datadoghq.com/logs/log_collection/javascript/